.github/CODEOWNERS

```diff
@@ -19,17 +19,25 @@
 # be requested for review when someone opens a pull request.
 # order is alphabetical for easier maintenance.
 #
+# Adam Ginna (adamginna-dell)
 # Alexander Hoppe (hoppea2)
 # Arindam Datta (dattaarindam)
 # Bahubali Jain (bahubali-jain)
+# Bartosz Ciesielczyk (cbartoszDell)
+# Bogdan Novikov (bogdanNovikovDell)
+# Florian Coulombel (coulof)
 # Francis Nijay (francis-nijay)
 # Harish P (harishp8889)
+# Leonard Ulman (Leonard-Dell)
+# Marcin Trajfacki (mtrajfacki-dell)
+# Marek Suski (mareksuski-dell)
+# Mateusz Urbanek (shanduur-dell)
+# Małgorzata Dutka (mdutka-dell)
 # Pooja Prasannakumar (kumarp20)
 # Prsanna Muthukumaraswamy (prablr79)
 # Shanmugapriya M (spriya-m)
 # Trevor Dawe (tdawe)
-# Florian Coulombel (coulof)
  
 
 # for all files:
-* @kumarp20 @bpjain2004 @francis-nijay @harishp8889 @spriya-m @prablr79 @hoppea2 @tdawe @coulof @shaynafinocchiaro @sharmilarama @isaiasA1 @arnchiequ-dell @alikdell @atye
+* @kumarp20 @bpjain2004 @francis-nijay @harishp8889 @spriya-m @prablr79 @hoppea2 @tdawe @coulof @shaynafinocchiaro @sharmilarama @isaiasA1 @arnchiequ-dell @alikdell @atye @adamginna-dell @cbartoszDell @bogdanNovikovDell @Leonard-Dell @mtrajfacki-dell @mareksuski-dell @shanduur-dell @mdutka-dell
```

Comment:
Can the unknown owner be removed from both lists?

Reply:
Please also remove Shanmugapriya M (spriya-m)